### PR TITLE
Make models mockable

### PR DESCRIPTION
**Pull request by ngastelum-ms**

### CODE REVIEW

**Review by drwill-ms — APPROVED** [review]

File: iothub/service/src/Registry/Models/ServiceStatistics.cs [hunk 1]
```diff
@@ -10,10 +10,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class ServiceStatistics
     {
+        /// <summary>
+        /// This constructor is for deserialization and unit test mocking purposes.
+        /// </summary>
```
Comment (brycewang-microsoft) [inline iothub/service/src/Registry/Models/ServiceStatistics.cs:15]: How about unifying the summaries across these classes?
```suggestion
        /// <summary>
        /// Creates an instance of this class. Provided for unit testing purposes only.
        /// </summary>
```

File: iothub/service/src/Registry/Models/RegistryStatistics.cs [hunk 2]
```diff
@@ -10,22 +10,31 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class RegistryStatistics
     {
+        /// <summary>
+        /// This constructor is for deserialization and unit test mocking purposes.
+        /// </summary>
```
Comment (brycewang-microsoft) [inline iothub/service/src/Registry/Models/RegistryStatistics.cs:15]: same here
```suggestion
        /// <summary>
        /// Creates an instance of this class. Provided for unit testing purposes only.
        /// </summary>
```

**Review by brycewang-microsoft — APPROVED** [review]

**Pull request merged**